crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -1188,6 +1188,8 @@ async fn mapper_publishes_child_device_create_message() {
 
     publish_a_fake_jwt_token(broker).await;
 
+    broker.publish("c8y/s/ds", "106,child-one").await.unwrap();
```

Comment:
Since the mapper init block is **not** waiting for the child device list response, is this really necessary?